Python/bltinmodule.c

```diff
@@ -1623,15 +1623,18 @@ hash as builtin_hash
     obj: object
     /
 
-Return the hash value for the given object.
+Return the integer hash value for the given object within this process.
 
 Two objects that compare equal must also have the same hash value, but the
-reverse is not necessarily true.
+reverse is not necessarily true.  Hash values may vary between Python
+processes.
+
+This hash value is used internally by Python dict and set hash tables.
```

Comment:
I'm not sure if this sentence adds value.

Reply:
Maybe remove the "hash tables" part.

Reply:
Or say "the dict and set builtin types".